oprheum-game/InstructionOverlayNode.swift

```diff
@@ -0,0 +1,90 @@
+//  InstructionOverlayNode.swift
+//
+//  Created by Clay Reimann
+//
+
+import SpriteKit
+
+class InstructionOverlayNode: SKNode {
+    let label1Height: CGFloat = 500
+    let label2Height: CGFloat = 300
+    let continueHeight: CGFloat = 50
+
+    var size = CGSize.zero {
+        didSet(oldSize) {
+            background.path = backgroundPath()
+            labelContinue.position = centeredPoint(at: continueHeight)
+            label1.position = centeredPoint(at: label1Height)
+            label2.position = centeredPoint(at: label2Height)
+        }
+    }
+
+    var text1 = "" {
+        didSet(oldText) {
+            label1.removeFromParent()
```

Comment:
This is to remove the old label if the text changes. It is necessary because of the way we display multi-line text: `label1` can actually be a `LabelNode` with many children who are also `LabelNode`s. This means that we can't simple set the text of `label1` to the new value, instead we have to create a new set of `LabelNode`s to display the new text.